js/src/modals/Shapeshift/Value/value.css

```diff
@@ -22,3 +22,9 @@
 .amount {
   display: inline-block;
 }
+
+.symbol {
+  font-variant: small-caps;
+  margin-left: 0.1rem;
+  font-size: 0.7rem;
```

Comment:
This should preferably be a percentage of the parent (e.g. `70%`). By doing this, you can easily change the font size of the parent without breaking the font size of `.symbol`.